dogstatsd/src/dogstatsd.rs

```diff
@@ -32,7 +32,7 @@ impl BufferReader {
         match self {
             BufferReader::UdpSocketReader(socket) => {
                 // TODO(astuyve) this should be dynamic
-                let mut buf = [0; 1024]; // todo, do we want to make this dynamic? (not sure)
+                let mut buf = [0; 8192]; // todo, do we want to make this dynamic? (not sure)
```

Comment:
Not sure about these two todos, but will you please leave a note about 8k matching the default of the Go agent?

Reply:
https://github.com/DataDog/libdatadog/blob/e2097f7963793f54e7c515740b89b88273fd8247/dogstatsd/src/dogstatsd.rs#L35

Reply:
Minor refactor following discussion with @alexgallotta 

https://github.com/DataDog/libdatadog/blob/e1cd0155f2d0e564319bc0ef21d91c2e0f49f358/dogstatsd/src/dogstatsd.rs#L35-L36